CHANGELOG.md

```diff
@@ -3,7 +3,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [1.1.1] - TBD
+## [2.0.0] - TBD
+### Changed
```

Comment:
Do we need to swap the order of `Changed` and `Added` to adhere to the Keep a Changelog?

Reply:
Nope we don't, will switch this back